### ISSUE TITLE
Maven dependency 2.0.x not available

### ISSUE THREAD

**Issue by Surajprakashsahu** [amzn/alexa-skills-kit-java#250]
I tried add ask-smapi-sdk version 2.x.x to  pom.xml got error that says: "Missing artifact com.amazon.alexa:ask-smapi-sdk:jar:2.0.0". 
```
<dependency>
	<groupId>com.amazon.alexa</groupId>
	<artifactId>ask-smapi-sdk</artifactId>
	<version>2.0.0</version>
</dependency>
```
Am I doing anything wrong?
[Link to maven ask-smapi-sdk](https://mvnrepository.com/artifact/com.amazon.alexa/ask-smapi-sdk)

Also there's no documentation available for version 1.x.
[Link to version 1.x ask-smapi-sdk ](https://github.com/alexa/alexa-skills-kit-sdk-for-java/tree/1.x/ask-smapi-sdk)

**Comment by pbheemag** [comment]
Hi @Surajprakashsahu ,
The latest version of `ask-smapi-sdk` is 1.0.1 which needs to be added as a dependency. Version 2.0.0 is not yet available. Hope this helps.

**Comment by Surajprakashsahu** [comment]
Thank You for the information.
But there's no source code or documentation available for version 1.x.
[404 error when tried to access version 1.x](https://github.com/alexa/alexa-skills-kit-sdk-for-java/tree/1.x/ask-smapi-sdk)

**Comment by pbheemag** [comment]
Hi @Surajprakashsahu ,
The documentation for `ask-smapi-sdk` 1.x can be found [here](https://github.com/alexa/alexa-skills-kit-sdk-for-java/tree/2.0.x/ask-smapi-sdk). The 2.0.x in the url doesn't represent the version of the `ask-smapi-sdk` but just a branch of the main repo. Let us know if you have further questions.

**Comment by Surajprakashsahu** [comment]
```
try {
    // response with just the body.
    Vendors response = customSmapiClient.listSkills();

    // For full response with header, body and status code.
    ApiResponse<Vendors> resp = customSmapiClient.callListSKills();
} catch (ServiceException e) {
    System.out.println(e.getMessage());
}
```

I tried above code from example, there's no method such as listSkills() and callListSKills() present in the Class SkillManagementService imported from "com.amazon.ask.model.services.skillManagement" package. Have I made any mistake here? Please help. 
Thank You.

**Comment by pbheemag** [comment]
The SDK exposes two methods which are similar to the name that you're trying to access, `listSkillsForVendorV1(params)` and `callListSkillsForVendorV1(params)`. May I know why you're using `listSkills()` and `callListSkills()`, and any documentation that might have misled you to use those methods?

**Comment by Surajprakashsahu** [comment]
I was getting started with this SDK for my application. I tried examples from the "Usage Example" from "Readme.md" of [ask-smapi-sdk](https://github.com/alexa/alexa-skills-kit-sdk-for-java/tree/2.0.x/ask-smapi-sdk) repository there I found those methods `listSkills()` and `callListSkills()`. 
Thank You for your help @pbheemag.
![image](https://user-images.githubusercontent.com/15258013/73386476-e8002b00-42f4-11ea-80a4-ca1c454e985d.png)
